cra-tf-validate-ignore-goals.json

```diff
@@ -0,0 +1,40 @@
+{
+    "scc_goals": [
+        {
+            "scc_goal_id": "3000107",
+            "description": "Check whether Cloud Object Storage network access is restricted to a specific IP range",
+            "ignore_reason": "This is part of the test example and not the current module",
+            "is_valid": false
+        },
+        {
+            "scc_goal_id": "3000114",
+            "description": "Check whether Cloud Object Storage buckets are enabled with IBM Activity Tracker",
+            "ignore_reason": "This is part of the test example and not the current module",
+            "is_valid": false
+        },
+        {
+            "scc_goal_id": "3000115",
+            "description": "Check whether Cloud Object Storage buckets are enabled with IBM Cloud Monitoring",
+            "ignore_reason": "This is part of the test example and not the current module",
+            "is_valid": false
+        },
+        {
+            "scc_goal_id": "3000116",
+            "description": "Check whether Cloud Object Storage bucket resiliency is set to cross region",
+            "ignore_reason": "This is part of the test example and not the current module",
+            "is_valid": false
+        },
+        {
+            "scc_goal_id": "3000408",
+            "description": "Check whether Flow Logs for VPC are enabled",
+            "ignore_reason": "This is part of the test example and not the current module",
+            "is_valid": false
+        },
+        {
+            "scc_goal_id": "3000452",
+            "description": "Check whether Virtual Private Cloud (VPC) network access control lists don't allow egress from 0.0.0.0/0 to any port",
+            "ignore_reason": "This is part of the test example and not the current module",
+            "is_valid": false
+        }
+    ]
+}
```

Comment:
What you have said in the ingore_reason is not true. The roks pattern deploys COS and VPC so all of these are valid results. I'll need to investigate each of them to decide what we do, so leave this with me